packages/suite-web/next.config.js

```diff
@@ -40,6 +39,14 @@ module.exports = withBundleAnalyzer(
                             ),
                         }),
                     );
+                    config.module.rules.push({
```

Comment:
not used, but I would keep it because we are probably gonna need some workers anyway (eg. exporting transactions csv)